docs/ContinuousIntegration.md

```diff
@@ -148,6 +148,14 @@ preset=buildbot_incremental,tools=RA,stdlib=RD,smoketest=macosx,single-thread
 
 ```
 
+### Testing Compiler Performance
+
+Platform        | Comment | Check Status
+------------    | ------- | ------------
+macOS platform  | @swift-ci Please test compiler performance       | Compiles full source compatibility test suite and measures compiler performance
+macOS platform  | @swift-ci Please smoke test compiler performance | Compiles a subset of source compatibility test suite and measures compiler performance
```

Comment:
This needs more elaboration. Do we describe what the source compatibility test suite is in this file? What do you mean by compiler performance? The counter stuff is significantly more than benchmarking. You should make it clear.

Reply:
I disagree that we should describe what it means at length here, that's not the purpose of this document. Linking to https://github.com/apple/swift/blob/master/docs/CompilerPerformance.md would perhaps be sufficient, what do you think?

Reply:
I wouldn't describe it at length. Instead what I would suggest is putting a blurb that says:

```
This measures general compiler performance by:

1. Running benchmarks.
2. Gathering counter information.
3. ...

For more information see <INSERT LINK HERE>.
```

Reply:
Better, can you move this /below/ the table and add the link. Once that is there, LGTM.